sgv2-restapi/pom.xml

```diff
@@ -262,6 +262,33 @@
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
       </plugin>
+      <!-- Plugin for building Docker images -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>jib-maven-plugin</artifactId>
+        <version>3.1.4</version>
+        <configuration>
+          <from>
+            <image>adoptopenjdk:8</image>
+          </from>
+          <to>
+            <image>docker.io/stargateio/stargate-restapi:${project.version}</image>
+          </to>
+          <!-- See: https://github.com/GoogleContainerTools/jib/tree/master/jib-maven-plugin#container-object-->
+          <container>
+            <creationTime>USE_CURRENT_TIMESTAMP</creationTime>
+            <filesModificationTime>${maven.build.timestamp}</filesModificationTime>
+            <ports>
+              <port>8082</port>
+            </ports>
+            <jvmFlags>
+              <jvmFlag>-Ddw.stargate.grpc.host=localhost</jvmFlag>
```

Comment:
On Grpc host: this would then build an image that expects gRPC to be running on same node, right?
Which works for testing, and some of smaller perf tests, but would not work when splitting. Unless we use something to do routing externally (service mesh / sidecar) and just bind via localhost?

Reply:
You'd need to override the value (which you'd now do via environment variable given the latest commit).